ibis/sql/mssql/compiler.py

```diff
@@ -0,0 +1,207 @@
+import pyodbc
+import sqlalchemy as sa
+import sqlalchemy.dialects.mssql as mssql
+
+
+import ibis.sql.alchemy as alch
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+import ibis.common.exceptions as com
+
+# used for literal translate
+from ibis.sql.alchemy import unary, fixed_arity
+
+
+def raise_unsupported_op_error(translator, expr, *args):
+    msg = "SQLServer backend doesn't support {} operation!"
+    op = expr.op()
+    raise com.UnsupportedOperationError(msg.format(type(op)))
+
+
+# Aggregation
+# coppied from postgresql compiler
+# support for of bit columns in aggregate methods
```

Comment:
can you import directly from the postgresql compiler (and/or move it to a generic function)

Reply:
I removed the comments since this method actually cannot use the postgresql method due to the sql function having a different name. But otherwise this method is very similar.